tests/test_resource_manager.py

```diff
@@ -0,0 +1,43 @@
+from contextlib import contextmanager
+from pathlib import Path
+from tests.base import TestCase, main # pylint: disable=import-error,no-name-in-module
+
+from pytest import fixture
+
+from ocrd_utils import pushd_popd, initLogging
+import ocrd_utils.constants
+
+_tempdir = None
+
+@contextmanager
+def monkey_patch_temp_xdg():
+    global _tempdir
+    with pushd_popd(tempdir=True) as tempdir:
+        old_config = ocrd_utils.constants.XDG_CONFIG_HOME
+        old_data = ocrd_utils.constants.XDG_DATA_HOME
+        ocrd_utils.constants.XDG_CONFIG_HOME = tempdir
+        ocrd_utils.constants.XDG_DATA_HOME = tempdir
+        from ocrd.resource_manager import OcrdResourceManager
+        yield tempdir, OcrdResourceManager()
+        ocrd_utils.constants.XDG_CONFIG_HOME = old_config
+        ocrd_utils.constants.XDG_DATA_HOME = old_data
+
+def test_config_created():
+    with monkey_patch_temp_xdg() as (tempdir, mgr):
+        f = Path(tempdir, 'ocrd', 'resources.yml')
+        assert f.exists()
```

Comment:
You probably meant 
```suggestion
        assert not f.exists()
```

Reply:
No, I was originally testing that the YAML file is created during instantiation. The problem is that any test or module that does `from ocrd import ...` will lead to the module being loaded before the XDG constant changing trickery happens. I.e. the test passes in isolation but not as part of the test suite. Only way to fix is to run the test separately, like we do for logging :/

Reply:
Oh, now I see